operator/pkg/manifest/helm_processor.go

```diff
@@ -124,12 +134,26 @@ func (h *helm) Install(stringifedManifest string, deployInfo types.InstallInfo,
 }
 
 // Uninstall transforms and deletes Helm based manifest using helm client.
-func (h *helm) Uninstall(stringifedManifest string, deployInfo types.InstallInfo, transforms []types.ObjectTransform,
+func (h *helm) Uninstall(stringifedManifest string, info types.InstallInfo, transforms []types.ObjectTransform,
 ) (bool, error) {
 	// convert for Helm processing
-	resourceLists, err := h.parseToResourceLists(stringifedManifest, deployInfo, transforms)
-	if err != nil {
-		return false, err
+	// do not retry since if the Kind is not existing anymore, there can never be a resource left in the cluster
+	// this means we can continue the installation
+	// resourceLists will not contain any resource.Infos that are not possible to use with the cluster
+	resourceLists, err := h.parseToResourceLists(stringifedManifest, info, transforms, false)
+
+	// here we verify that out of all errors, the error that we are seeing here is actually an aggregate
+	// this is because for every resource there can be at least one error
+	// however due to being an uninstallation, we can continue for NoKindMatchErrors.
+	// thus, if all errors are of type meta.NoKindMatchError, we can simply continue the uninstallation.
+	var multiErr *types.MultiError
+	if ok := errors.As(err, &multiErr); ok {
+		for i := range multiErr.Errs {
+			var noMatchErr *meta.NoKindMatchError
```

Comment:
```suggestion
                var noMatchErr *meta.NoKindMatchError
		for i := range multiErr.Errs {
			
```

Reply:
this is a pointer and thus I think is safer to reinitialise as after one is written, and errors.As would error, it would still contain the old pointer value (of a previous loop iteration)